tests/unit/globals/formatting.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { formatNumber } from '@/globals/formatting'
+
+describe('formatters', () => {
+  describe('format number', () => {
+    it('should return a 0 when a negative value is given', () => {
```

Comment:
this case is not really formatting

Reply:
it formats all negatives to a zero, but again debatable.